src/common/options.rs

```diff
@@ -112,13 +109,43 @@ impl Endpointing {
     #[allow(missing_docs)]
     pub fn as_str(&self) -> String {
         match self {
-            Endpointing::True(_value) => "true".to_string(),
-            Endpointing::False(_value) => "false".to_string(),
+            Endpointing::Enabled(value) => {
+                if *value {
+                    "true".to_string()
+                } else {
+                    "false".to_string()
+                }
+            },
```

Comment:
```suggestion
            Endpointing::Enabled(value) => value.to_string()
```

This can be simplified. See [this playground example](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021) to show how booleans are converted.